superset-frontend/src/explore/components/controls/AnnotationLayerControl/AnnotationLayer.jsx

```diff
@@ -726,6 +738,14 @@ export default class AnnotationLayer extends React.PureComponent {
                 value={!show}
                 onChange={v => this.setState({ show: !v })}
               />
+              <CheckboxControl
+                name="annotation-label-show"
+                label={t('Show Label')}
```

Comment:
This is a nit, I think our convention is` Show label `rather than `Show Label`

```suggestion
                label={t('Show label')}
```